src/components/cc-feature-list/cc-feature-list.js

```diff
@@ -0,0 +1,162 @@
+import '../cc-badge/cc-badge.js';
+import '../cc-block/cc-block.js';
+import '../cc-link/cc-link.js';
+import '../cc-loader/cc-loader.js';
+import '../cc-notice/cc-notice.js';
+import '../cc-toggle/cc-toggle.js';
+
+/**
+ * @typedef {import('./cc-feature-list.types.d.ts').Feature} Feature
+ * @typedef {import('./cc-feature-list.types.d.ts').FeatureListState} FeatureListState
+ * @typedef {import('./cc-feature-list.types.d.ts').FeatureListStateLoading} FeatureListStateLoading
+ * @typedef {import('./cc-feature-list.types.d.ts').FeatureListStateLoaded} FeatureListStateLoaded
+ * @typedef {import('./cc-feature-list.events.js').CcFeatureValueChangeEvent} CcFeatureValueChangeEvent
+ */
+
+/**
+ * A component to display a list of feature flag and to manage their activation.
+ *
+ * @cssdisplay block
+ */
+export class CcFeatureList extends LitElement {
+  static get properties() {
+    return {
+      state: { type: Object },
+    };
+  }
+
+  constructor() {
+    super();
+
+    /** @type {FeatureListState} Set the state of the component */
+    this.state = { type: 'loading' };
+  }
+
+  /**
+   * @param {Feature} feature
+   * @param {String} newValue
+   */
+  _onToggleChange(feature, newValue) {
+    if (this.state.type === 'loading') {
+      return;
+    }
+
+    this.dispatchEvent(
+      new CustomEvent('cc-feature-setting-change', {
+        detail: {
+          featureId: feature.featureId,
+          newValue,
+        },
+        bubbles: true,
+        composed: true,
+      }),
+    );
+
+    console.log(newValue);
+  }
+
+  /**
+   * @param {Feature} feature
+   */
+  renderFeature(feature) {
+    return html`
+      <cc-block class="feature">
+        <div slot="header-title">
+          ${feature.featureName}
+          ${feature.status
+            ? html`
+                <cc-badge intent="${feature.status === 'alpha' ? 'warning' : feature.status === 'beta' ? 'info' : ''}">
+                  ${feature.status}
+                </cc-badge>
+              `
+            : ''}
+        </div>
+        <div slot="content">${feature.featureDescription}</div>
+        <div slot="footer">
+          ${feature.documentationLink
+            ? html`
+                <cc-link .href=${feature.documentationLink} a11y-desc="Go to documentation page" .icon=${iconInfo}>
+                  ${i18n('cc-feature-list.documentation-link')}
+                </cc-link>
+              `
+            : ''}
+          ${feature.feedbackLink
+            ? html`
+                <cc-link .href=${feature.feedbackLink} a11y-desc="Go to feedback page" .icon=${iconMessage}>
+                  ${i18n('cc-feature-list.feedback-link')}
+                </cc-link>
+              `
+            : ''}
+        </div>
+        <div slot="header-right">
+          <cc-toggle
+            .choices="${feature.toggleOptions}"
+            value="${feature.toggleValue}"
+            @cc-select=${(/** @type {CcFeatureValueChangeEvent} */ event) =>
```

Comment:
fix: wrong type `CcFeatureValueChangeEvent`. Here you receive the event dispatched by `cc-select` component which is a `CcSelectEvent`